lib/sprockets/commoner/processor.rb

```diff
@@ -54,6 +55,17 @@ def initialize(root, include: [root], exclude: ['vendor/bundle'], babel_exclude:
         super(root, 'NODE_PATH' => JS_PACKAGE_PATH)
       end
 
+      def cache_key
+        @cache_key ||= [
+          self.class.name,
+          VERSION,
+          version,
+          @include.map(&:to_s),
+          @exclude.map(&:to_s),
+          @babel_exclude.map(&:to_s),
+        ]
+      end
```

Comment:
This is only on the instance as we won't have the full configuration until the instance is created (which we can't unless we have an environment). The singleton processor will just have a `nil` cache_key